subsys/bluetooth/mesh/Kconfig

```diff
@@ -59,12 +59,16 @@ choice BT_MESH_ADV
 
 menuconfig BT_MESH_ADV_LEGACY
 	bool "Legacy advertising"
+	depends on BT_LL_SW_SPLIT
 	help
 	  Use legacy advertising commands for mesh sending. Legacy
-	  advertising is significantly slower than the extended advertising, but
-	  is supported by all controllers.
+	  advertising is significantly slower than the extended advertising.
 
-	  WARNING: The legacy advertiser can occasionally do more message
+	  WARNING: This feature is no longer supported in NCS. The
+	  legacy advertiser does not work with SDC, so the Zephyr Link Layer
+	  must be used.
+
+	  ADDITIONAL WARNING: The legacy advertiser can occasionally do more message
```

Comment:
Is this the syntax requirement to write "ADDITIONAL WARNING"?

Reply:
Couldn't find another example of two separate warnings, I can just do a newline instead adding to the existing warning?

Reply:
I'd suggest to leave just "WARNING" if syntax allows.